### DISCUSSION TITLE
Empty state Page- 'Results not found'

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#1021] [Ideas] [answered]
I'm working on an empty state page for Phone numbers and I wanted to check in with you what would be the best format. 
I have 2 options (1 with a card and the other as text below the search results:  Also, i wanted to ask you what is the Anchor option when the link is outside the Console. 

![Captura de Pantalla 2020-12-09 a la(s) 5 06 21 p  m](https://user-images.githubusercontent.com/72419222/101694342-f8b71c80-3a40-11eb-8636-153876624d77.png)
![Captura de Pantalla 2020-12-09 a la(s) 5 06 31 p  m](https://user-images.githubusercontent.com/72419222/101694347-fa80e000-3a40-11eb-960d-49c23f828308.png)
![Captura de Pantalla 2020-12-09 a la(s) 5 06 42 p  m](https://user-images.githubusercontent.com/72419222/101694350-fb197680-3a40-11eb-9654-7ed60a5b5b47.png)

**Comment by eleifer-twilio** [comment]
I have a similar design needed. Please let me know what is decided here! :) @lgalloherran @serifluous

**Comment by TheSisb** [comment, marked as answer]
Hi @lgalloherran ,

1- For external icon, we have a `showExternal` prop on our Anchor component: https://paste.twilio.design/components/anchor/#external-anchor Easy! :)

2- For empty state, we don't have a pattern yet but we're looking for contributors: https://paste.twilio.design/patterns/empty-state/
